services/venice-server/src/main/java/com/linkedin/venice/stats/ServerHttpRequestStats.java

```diff
@@ -101,31 +95,36 @@ public ServerHttpRequestStats(
         "storage_engine_query_latency",
         totalStats,
         () -> totalStats.databaseLookupLatencySensor,
-        TehutiUtils.getPercentileStat(getName(), getFullMetricName("storage_engine_query_latency")),
-        new Avg(),
-        new Max());
+        TehutiUtils.get99PercentileStatWithAvgAndMax(getName(), getFullMetricName("storage_engine_query_latency")));
     databaseLookupLatencyForSmallValueSensor = registerPerStoreAndTotal(
         "storage_engine_query_latency_for_small_value",
         totalStats,
         () -> totalStats.databaseLookupLatencyForSmallValueSensor,
-        TehutiUtils.getPercentileStatWithAvgAndMax(
+        TehutiUtils.get99PercentileStatWithAvgAndMax(
             getName(),
             getFullMetricName("storage_engine_query_latency_for_small_value")));
     databaseLookupLatencyForLargeValueSensor = registerPerStoreAndTotal(
         "storage_engine_query_latency_for_large_value",
         totalStats,
         () -> totalStats.databaseLookupLatencyForLargeValueSensor,
-        TehutiUtils.getPercentileStatWithAvgAndMax(
+        TehutiUtils.get99PercentileStatWithAvgAndMax(
             getName(),
             getFullMetricName("storage_engine_query_latency_for_large_value")));
 
-    storageExecutionHandlerSubmissionWaitTime = registerSensor(
+    storageExecutionHandlerSubmissionWaitTime = registerOnlyTotalSensor(
         "storage_execution_handler_submission_wait_time",
-        TehutiUtils.getPercentileStatWithAvgAndMax(
+        totalStats,
+        () -> totalStats.storageExecutionHandlerSubmissionWaitTime,
+        TehutiUtils.get99PercentileStatWithAvgAndMax(
             getName(),
             getFullMetricName("storage_execution_handler_submission_wait_time")));
 
-    storageExecutionQueueLenSensor = registerSensor("storage_execution_queue_len", new Max(), new Avg());
+    storageExecutionQueueLenSensor = registerOnlyTotalSensor(
+        "storage_execution_queue_len",
+        totalStats,
+        () -> totalStats.storageExecutionQueueLenSensor,
+        new Max(),
+        new Avg());
```

Comment:
For `storage_execution_queue_len`, we plan to remove `Max` and `Avg` metrics, don't we?